app/templates/components/navigation/scroll-top-button.html

```diff
@@ -0,0 +1,11 @@
+<a
+    id="scroll-top-button"
+    href="#app"
+    class="absolute right-0 bottom-0 bg-corporate-blue h-13 py-4 border-l-2 border-white translate-x-0 translate-x-0 ease-in duration-100 {% if false %}translate-x-full{% endif %}"
+>
+    <img
+        class="w-5 -rotate-90 mx-8"
+        src="{{ url_for('static', path='images/arrow-right.svg') }}"
+        alt="{{ translations.nav.alignIconAlt }}"
```

Comment:
This `alt` is a bit misleading, because it will say `"Inhaltsverzeichnis"`, even though this link does something completely different. You could consider adding something like `aria-label="Zum Start der Seite gehen"` to the `<a>` and removing the `alt` tag here.